spec/regional_rubykaigi_org_spec.rb

```diff
@@ -1,6 +1,14 @@
 require_relative "./spec_helper"
 
 describe "http://regional.rubykaigi.org" do
+  describe "(http) /" do
+    let(:res) { http_get("http://regional.rubykaigi.org/") }
+    it "redirects to https" do
+      expect(res.code).to eq("302")
```

Comment:
```suggestion
      expect(res.code).to eq("301")
```

httpsだぞ! ということなら `301` のほうがいいのかな?

Reply:
301 ですね